composer.json

```
@@ -9,20 +9,20 @@
         "sylius/sylius": "^1.8@dev"
     },
     "require-dev": {
-        "behat/behat": "^3.4",
+        "behat/behat": "^3.6.1",
         "behat/mink": "^1.7@dev",
```

```suggestion
        "behat/mink": "^1.8",
```
As in https://github.com/Sylius/Sylius-Standard/pull/404/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780R28